### PR TITLE
docs(tabs): add demo for custom html

### LINKED ISSUE #3548: docs(tabs): add custom content to demo
Add section Custom content to demo page (according to #3267)

### PR DESCRIPTION
**Pull request by artolshansky**
close #3548

# PR Checklist
Before creating new PR, please take a look at checklist below to make sure that you've done everything that needs to be done before we can merge it.

 - [x] read and followed the [CONTRIBUTING.md](https://github.com/valor-software/ngx-bootstrap/blob/development/CONTRIBUTING.md) guide.
 - [x] built and tested the changes locally.
 - [x] added/updated demos.

### CODE REVIEW

File: demo/src/app/components/+tabs/demos/custom-template/custom-template.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div>
+  <tabset>
+    <tab heading="Tab 1">Tab content</tab>
+    <tab (select)="alertMe()">
```
Comment (IlyaSurmay) [inline demo/src/app/components/+tabs/demos/custom-template/custom-template.html]: perhaps there should be a bit more tabs with custom templates. for example, first is triggering alert, second has some html tags in it, third has an icon

Comment (artolshansky) [reply demo/src/app/components/+tabs/demos/custom-template/custom-template.html]: changed

**Review by IlyaSurmay — DISMISSED** [review]

File: demo/src/app/components/+tabs/demos/custom-template/custom-template.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<div>
+  <tabset>
+    <tab heading="Alert!" (select)="alertMe()">
```
Comment (EvilAlexei) [inline demo/src/app/components/+tabs/demos/custom-template/custom-template.html]: One demo - one feature. Please remove `select` method

**Review by EvilAlexei — CHANGES_REQUESTED** [review]

**Review by EvilAlexei — DISMISSED** [review]

**Review by EvilAlexei — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Why changes in Basic example are needed for demo with custom html?

**Comment by artolshansky** [comment]
@YevheniiaMazur in this PR didn't needed, switched it back

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — DISMISSED** [review]

**Pull request merged**